bamboo/bootstrap-lint-audit.sh

```diff
@@ -12,6 +12,8 @@ if [[ $USE_CACHED_BOOTSTRAP == true ]]; then ## Change into cached cumulus, pull
   git fetch --all
   git checkout "$GIT_SHA"
   rm -f package-lock.json
+else
```

Comment:
This else probably isn't needed, just remove the git fetch --all from both conditionals and drop it below the if loop.  In either case it should be in the proper directory.

Reply:
Scratch that, as discussed elsewhere it may cause issues with the git checkout.